mypy_django_plugin/django/context.py

```diff
@@ -60,12 +60,11 @@ def initialize_django(settings_module: str) -> Tuple["Apps", "LazySettings"]:
         from django.apps import apps
         from django.conf import settings
 
-        apps.get_models.cache_clear()  # type: ignore
         apps.get_swappable_settings_name.cache_clear()  # type: ignore
+        apps.clear_cache()
```

Comment:
Btw, why `apps.get_swappable_settings_name.cache_clear()` is not a part of `app.clear_cache()`? Should I commit this to Django?

Reply:
Make a ticket!

Reply:
Done https://code.djangoproject.com/ticket/34687#ticket